src/app/dashboard/hex-map/hex-map.component.ts

```diff
@@ -19,12 +20,26 @@ import topicDefinitions from "../../../assets/static/json/topicDefinitions.json"
 export class HexMapComponent implements OnInit, OnDestroy {
     @ViewChild('deckContainer', {static: true}) deckContainer!: ElementRef;
 
-    private relevantState = computed(() => this.stateService.appState())
+    private relevantState = computed((): IStateParams => {
+        return this.stateService.appState()
+    }, {
+        equal: (a, b) => {
+            return a.hashtag === b.hashtag
+                && a.start === b.start
+                && a.end === b.end
+                && a.topics == b.topics
+                && a.countries == b.countries
+        }
+    })
 
     selectedTopic!: StatsType;
     deck!: Deck;
     minMaxStats!: { result: { max: number; min: number } };
     private layer!: H3HexagonLayer<HexDataType>;
+    currentResolution = 3;
+    private MAX_HEX_CELL = 314000;
```

Comment:
how did you arrive at that number, is it based on deckgl limitations or simply a reasonably big number?